src/node/utils.ts

```diff
@@ -11,3 +11,7 @@ export const createHash = (info: string): string => {
   }
   return createHashFunc('sha256').update(info).digest('hex').slice(0, 8);
 };
+
+export const toSeq = (url: string, sep = '/') => {
```

Comment:
How about rename it to `normalizePath`? It will be easier to understand.

Reply:
Sounds good

Reply:
> 怎么改名`normalizePath`？会更容易理解。

Sounds good